File: leetcode in c/average value of even number.c
int averageValue(int* nums, int numsSize) {
    int sum = 0;
    int count = 0;
    for (int i = 0; i < numsSize; i++)
    {
        if (nums[i] % 6 == 0)
        {
            sum += nums[i];
            count++;
        }
    }
    if (count == 0)
    {
        return 0;
    }
    return round(sum / count);
}

File: leetcode beginners/average value of even number.py
class Solution:
    def averageValue(self, nums: List[int]) -> int:
        arr = []
        count = 0
        for i in range(len(nums)):
            if nums[i] % 6 == 0:
                arr.append(nums[i])
                count += 1
        if count == 0:
            return 0
        return sum(arr) // len(arr)